docs/api-documentation.md

```diff
@@ -0,0 +1,370 @@
+# API Documentation
+
+This document provides a brief overview of the different API endpoints, their usage and purpose.
+
+## Authentication
+
+This section lists the authentication related requests such as login.
+
+### Login
+
+Allows the user to login into the portal and receive the auth token from the API. Requires the user email and password values.
+
+**User Login Flow**
```

Comment:
I think we can use #### heading for this instead of bold. This is a heading.

Reply:
fixed